spec/fixtures/converter/masscan.json

```diff
@@ -0,0 +1,3 @@
+[
+  { "ip": "93.184.216.34", "timestamp": "1629960621", "ports": [ {"port": 80, "proto": "tcp", "status": "open", "reason": "syn-ack", "ttl": 54} ] }
```

Comment:
Fixture file should have at least two values. You could copy another value from `spec/fixtures/masscan.json`.